brave/brave-core
# Adds option to reset rewards

Resolves https://github.com/brave/brave-browser/issues/10064

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:
- enable rewards
- click on gear icon
- click on reset tab and click reset
- make sure that you see confirmation
- make sure that both rewards page and panel gets reset to join rewards screen
- go to profile and make sure that `publisher_info_db` file, `rewards_service` folder and `ads_service` folder are removed
- to to Preference file and make sure that `brave.rewards` and `brave.ads` are reset

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
